jassign/to_ok.py

```diff
@@ -362,7 +393,18 @@ def gen_views(master_nb, result_dir, endpoint, no_submit_cell):
     trash_dir = result_dir / 'trash'
     os.makedirs(autograder_dir, exist_ok=True)
     ok_nb_path = convert_to_ok(master_nb, autograder_dir, endpoint, no_submit_cell)
+    
     shutil.copytree(autograder_dir, student_dir)
+    if open_ag_tests:
+        # In the autograder view, use the test files in the open_tests/ directory.
+        shutil.rmtree(autograder_dir / "tests")
+        shutil.move(autograder_dir / "open_tests", autograder_dir / "tests")
```

Comment:
Why generate both if you're just going to delete one? The same behavior could be implemented by just passing a hidden value into `gen_case` rather than duplicating everything.